lib/schedule/minischedule/load.ex

```diff
@@ -95,17 +121,93 @@ defmodule Schedule.Minischedule.Load do
     }
   end
 
-  defp operator_to_piece(activity, _trips) do
-    activity
-  end
-
   @spec trip_in_operator(Activity.t(), Trip.t()) :: boolean()
   defp trip_in_operator(%Activity{activity_type: "Operator"} = activity, trip) do
     trip.start_time >= activity.start_time and
       trip.end_time <= activity.end_time and
       String.contains?(trip.block_id, activity.partial_block_id)
   end
 
+  @spec operator_is_as_directed?(Activity.t()) :: boolean()
+  defp operator_is_as_directed?(%Activity{activity_type: "Operator"} = activity) do
+    String.contains?(activity.partial_block_id, "ad")
```

Comment:
@skyqrose this makes me a little leery but I'm assuming that if you did it this way there's probably not a better way to distinguish as-directed activities.

Reply:
Here's an activity for scheduled trips:
```
aba20011;123;    1504;05:39;09:37;albny;kenbs;Operator; 60 - 52
```
and here's one for wad
```
abc20011;123;    1151;20:30;23:30;cabot;cabot;Operator;wad -342
```
so no, not really, that's the only difference.